Automattic/wp-calypso
# Social: Fix pricing page bug after buying the advanced plan

#### Proposed Changes

* Refactored how the social plans are rendered.
* Right, now if you buy a Jetpack Social Advanced plan, the pricing page shows you and lets you buy the basic plan. We should follow the same approach as the Backup plans and show only the plan you own. 
* Added Jetpack Social Advanced to the list of Jetpack Social Products.

#### Testing Instructions



* Boot up a JN site and buy Jetpack Social Advance on it. You can buy the advanced plan by going to the pricing page. 
* Go to your Jetpack Dashboard and click on plans. This will take you to the plans page, where you see the bug as shown below 
<img width="1720" alt="Screenshot 2023-02-01 at 18 26 39" src="https://user-images.githubusercontent.com/6594561/216049212-752db0d7-6edf-4bd0-bae9-ea49ee0fdab7.png">

* Switch to this branch on local and do `yarn start-jetpack-cloud`
* Go to the Jetpack page's view on your localhost or live site by appending `pricing/my-locals.test.site?site=my-locals.test.site?` where `my-locals.test.site` is the URL of your test site without the https protocol and ensure that only see one jetpack social product, i.e advanced or social with a manage subscription button next to it 
<img width="1329" alt="Screenshot 2023-02-01 at 18 30 40" src="https://user-images.githubusercontent.com/6594561/216050179-2067e20f-273c-4bfe-8735-a89b9abce98a.png">


#### Pre-merge Checklist





Related to #
